tests/test_datasets/test_dataset.py

```diff
@@ -77,3 +77,16 @@ def test_dataset_datetime_convertion_during_init():
     exog.index = df.index.astype(str)
     ts = TSDataset(df, "1d", exog)
     assert ts.df.index.dtype == "datetime64[ns]"
+
+
+def test_make_future_small_horizon():
+    timestamp = np.arange(np.datetime64("2021-01-01"), np.datetime64("2021-02-01"))
+    target1 = [np.sin(i) for i in range(len(timestamp))]
+    target2 = [np.cos(i) for i in range(len(timestamp))]
+    df1 = pd.DataFrame({"timestamp": timestamp, "target": target1, "segment": "1"})
+    df2 = pd.DataFrame({"timestamp": timestamp, "target": target2, "segment": "2"})
+    df = pd.concat([df1, df2], ignore_index=True)
+    df = TSDataset.to_dataset(df)
+    ts = TSDataset(df, freq="D")
+    train = TSDataset(ts[: ts.index[10], :, :], freq="D")
+    assert len(train.make_future(1).df) == 1
```

Comment:
Shouldn't we expect warning here? https://docs.pytest.org/en/latest/how-to/capture-warnings.html#warns